src/components/content/order/migrate/MigrateServiceStatusPolling.tsx

```diff
@@ -0,0 +1,150 @@
+    isLoading: boolean;
+    setIsDestroying: (arg: boolean) => void;
+    setIsDestroyingCompleted: (arg: boolean) => void;
+}): React.JSX.Element {
+    const getDeployedServiceDetailsByIdQuery = useServiceDetailsPollingQuery(uuid, [
+        ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
+        ServiceDetailVo.serviceDeploymentState.DESTROY_SUCCESS,
+    ]);
+
+    const stopWatch = useStopwatch({
+        autoStart: true,
+    });
+
+    useEffect(() => {
+        if (
+            getDeployedServiceDetailsByIdQuery.data &&
+            [
+                ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
+                ServiceDetailVo.serviceDeploymentState.DESTROY_SUCCESS,
+            ].includes(getDeployedServiceDetailsByIdQuery.data.serviceDeploymentState)
+        ) {
+            setIsDestroying(false);
+            setIsDestroyingCompleted(true);
+        }
+    }, [getDeployedServiceDetailsByIdQuery.data, setIsDestroyingCompleted, setIsDestroying]);
+
+    useEffect(() => {
+        if (error) {
+            setIsDestroying(false);
+            setIsDestroyingCompleted(true);
+        }
+    }, [error, setIsDestroyingCompleted, setIsDestroying]);
+
+    useEffect(() => {
+        if (getDeployedServiceDetailsByIdQuery.error) {
+            setIsDestroying(false);
+            setIsDestroyingCompleted(true);
+        }
+    }, [getDeployedServiceDetailsByIdQuery.error, setIsDestroyingCompleted, setIsDestroying]);
+
+    if (isLoading) {
+        return OrderSubmitResult(
+            'Request submission in-progress',
+            uuid,
+            'success',
+            ServiceDetailVo.serviceDeploymentState.DESTROYING,
+            stopWatch,
+            OperationType.Destroy
+        );
+    }
+
+    if (error) {
+        return OrderSubmitFailed(
+            error,
+            ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
+            stopWatch,
+            OperationType.Destroy
+        );
+    }
+
+    if (uuid && getDeployedServiceDetailsByIdQuery.error) {
+        return OrderSubmitResult(
+            'Destroy status polling failed. Please visit MyServices page to check the status of the request.',
+            uuid,
+            'error',
+            ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
+            stopWatch,
+            OperationType.Destroy
+        );
+    }
+
+    if (uuid && getDeployedServiceDetailsByIdQuery.data)
+        if (
+            ![
+                ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
+                ServiceDetailVo.serviceDeploymentState.DESTROY_SUCCESS,
+            ].includes(getDeployedServiceDetailsByIdQuery.data.serviceDeploymentState)
+        ) {
+            return OrderSubmitResult(
+                'Migrating, Please wait...',
+                uuid,
+                'success',
+                ServiceDetailVo.serviceDeploymentState.DESTROYING,
+                stopWatch,
+                OperationType.Destroy
```

Comment:
if OperationType is Destroy.. then we will show 'Destoryed Succesfully'. I think we need OperationType as 'Migrate' and add messages according to that.